SCClassLibrary/Common/GUI/Base/Grid.sc

```diff
@@ -106,6 +53,14 @@ DrawGrid {
 	vertGrid_ { arg g;
 		y.grid = g;
 	}
+	tickSpacing_ { arg xpx, ypx;
+		x.tickSpacing = xpx;
+		y.tickSpacing = ypx;
+	}
+	numTicks_ { arg numx, numy;
+		x.numTicks = numx;
+		y.numTicks = numy;
+	}
```

Comment:
The main motivation was that having the convenience method `tickSpacing_` that sets both axes to the _same_ spacing doesn't make much sense considering the scales of the data is usually different on each axis, as well as the aspect ratio. Same for `numTicks_`.

Another advantage is not having to think about individual axes as separate objects.

Reply:
I agree, because this would show in the code _that_ you can access these objects separately.

Reply:
> The main motivation was that having the convenience method tickSpacing_ that sets both axes to the same spacing doesn't make much sense

@mtmccrea yes, I get that. But my point was do we really need that convenience method at all, if we can do tickSpacing on individual grids... which you address in the second paragraph.

@telephon I'm not sure whether you agree with adding the convenience method or not adding it? :)

Reply:
Thanks for considering this, @dyfer, 
>do we really need that convenience method at all, if we can do tickSpacing on individual grids... which you address in the second paragraph.

In addition to those reasons I gave, another reason is that there are similar convenience methods for multiple other properties of `DrawGridX/Y` (bounds, font, fontColor, gridColors). So in the interest of a complete implementation, again which models the behavior of `Plotter`, I'd say yes.

Reply:
ok, well then that is a good reason to have them. It may not have been a good idea (by me) to add these to `Plotter` in the first place, but well :) …

Reply:
Actually I appreciate being able to set the properties of all subplots from `Plotter` with arrays :)

Reply:
:)